Exercise_6/Exercise_6.ipynb

```diff
@@ -281,7 +318,7 @@
     "\n",
     "Let's examine the difference between git fetch and git pull.  We just used git fetch to get a commit from our remote repository, and then we used git merge to include it in our local branch.  \n",
     "\n",
-    "First, go back to the web interface and use it to make a new commit.  \n",
+    "First, go back to the web interface and use it to make a new commit by editing a file.  \n",
```

Comment:
In which branch should the user be? It confused me a little. Also I managed to get a merge conflict like this, so the description should be so that this will not happen?